test/integration/count_test.lua

```diff
@@ -26,6 +27,9 @@ pgroup.before_all(function(g)
     g.cluster:server('router').net_box:eval([[
         require('crud').cfg{ stats = true }
     ]])
+    g.cluster:server('router').net_box:eval([[
+        require('crud.ratelimit').disable()
```

Comment:
What do we need it for?

Reply:
It disable the ratelimit for tests:

https://github.com/tarantool/tarantool/pull/7131#pullrequestreview-972405738
https://github.com/tarantool/tarantool/pull/7131#pullrequestreview-973719435